fdbclient/ReadYourWrites.actor.cpp

```diff
@@ -1391,13 +1396,15 @@ void ReadYourWritesTransaction::updateConflictMap( KeyRangeRef const& keys, Writ
 	for(; it.beginKey() < keys.end; ++it ) {
 		if( it.is_unmodified_range() || ( it.is_operation() && !it.is_independent() ) ) {
 			KeyRangeRef insert_range = KeyRangeRef( std::max( keys.begin, it.beginKey().toArenaOrRef( arena ) ), std::min( keys.end, it.endKey().toArenaOrRef( arena ) ) );
-			if( !insert_range.empty() )
+			if (!insert_range.empty()) {
+				approximateSize += keys.expectedSize() + sizeof(KeyRangeRef);
 				readConflicts.insert( insert_range, true );
+			}
 		}
 	}
 }
 
-void ReadYourWritesTransaction::writeRangeToNativeTransaction( KeyRangeRef const& keys ) {
+void ReadYourWritesTransaction::writeRangeToNativeTransaction(KeyRangeRef const& keys, bool updateSize) {
```

Comment:
I think all callers to this function won't want to update the size, but instead we will want this argument to be present on `writeConflictRange`, with calls from this function passing false.

Reply:
I see. Everything here is already counted, so no need to count again.  The `addWriteConflictRange` call within this function is calling native transaction's method, which also do not need to count again.